conf/test/titan/grakn.properties

```diff
@@ -108,6 +108,7 @@ loader.threads=8
 #Authentication
 password.protected=false
 JWT.secret=Anzdz9l4TG8N5y18oaTDueqJQca1aX7loaz0U8Hz
+admin.password=grakn
```

Comment:
There is one configuration file that is missing this new property: `conf/test/orientdb/grakn.properties`.